src/libAtomVM/interop.c

```diff
@@ -229,3 +229,18 @@ term interop_map_get_value_default(Context *ctx, term map, term key, term defaul
         return term_get_map_value(map, pos);
     }
 }
+
+int interop_atom_term_select_int(GlobalContext *global, const AtomStringIntPair *table, term atom)
+{
+    if (UNLIKELY(!term_is_atom(atom))) {
+        return false;
```

Comment:
Should this be something like -1?  Or pass in `bool *ok`?

Reply:
Good catch, I'm going to remove it:
`globalcontext_is_term_equal_to_atom_string` doesn't fail when given argument is not a term atom.
It will be a bit slower for sure, since it needs to compare n times before returning, but it will be bug free.

The behavior I would expect here is returning the default value, the one associated to NULL.